frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -925,6 +926,8 @@ export function MaterializeVolumeAnnotationModal({
         output dataset and the output segmentation layer.
       </p>
     );
+  } else {
+    includesProofreading = tracing.volumes.some((v) => v.hasEditableMapping === true);
```

Comment:
If my annotation contains two volume layers and only one has an editable mapping, that property will be true. However, i I want to export the layer that has no editable mapping, `true` will still be used in this UI and sent to the backend, which seems wrong?